crates/astria-composer/src/searcher/mod.rs

```diff
@@ -52,10 +52,13 @@ mod rollup;
 
 use collector::Collector;
 
-/// the astria seqeuencer.
+/// A Searcher collates transactions from multiple rollups and bundles them into
+/// Astria sequencer transactions using arbitrary strategies and then passed on to the 
+/// shared sequencer. These sequencer transactions have the highly desirable quality of
+/// atomic inclusion, i.e. if they are submitted to the sequencer, all of them are going
+/// to be executed in the same Astria block.
```

Comment:
Want to avoid marketing language here ie `highly desirable quality`.

We also don't currently have arbitrary strategies.